docs/pages/getting-started/digitalocean.mdx

```diff
@@ -0,0 +1,117 @@
+
+![Setup FQDN](../../img/quickstart/digitalocean/fqdn.png)
+
+## Step 2/4. Configure Teleport
+When you are ready with your FQDN, SSH to your droplet, a setup wizard will guide you through the initial Teleport set up in your first SSH login to the droplet.
+
+```
+$ ssh root@teleport_host
+*********************************************************
+**              Configuring Teleport                   **
+*********************************************************
+___
+• Enter Teleport cluster name (FQDN): dodemo.teleporters.dev
+• Enter your email address to retrieve TLS certificate from Lets Encrypt: dodemo@teleporters.dev
+• Enter a username for the initial Teleport user: tadmin
+
+Initializing...
+[+] Generating new host UUID...
+[+] Updating cluster networking configuration...
+[+] Generating user and host certificate authority...
+[+] Enabling RBAC in OSS Teleport. Migrating users, roles and trusted clusters...
+[+] Starting Auth and Proxy services...
+[+] Final Checks...
+
+***************************************************************************
+**                             ---
+**    Teleport is configured and user tadmin has been created
+**    but requires a password. Open the URL link below to complete the
+**    setup. The link is valid for 1h:
+**
+**    https://dodemo.teleporters.dev:443/web/invite/992a29e8ef845d2f0189dee316f92939
+**
+**                             ---
+**                     HAPPY TELEPORTING :)
+***************************************************************************
+```
+
+Copy the URL link printed in this step:
+`https://dodemo.teleporters.dev:443/web/invite/992a29e8ef845d2f0189dee316f92939`
+
+This link opens up Teleport Web UI, where you will need to set a password and configure two-factor authentication to complete the user setup process.
+
+## Step 3/4. Complete user setup and login to Teleport UI
+Open the link copied in the previous step in the browser to complete the setup process. When the web page is ready:
+1. Scan the QR code with two-factor authentication app (e.g., Google Authenticator)
+2. Set a password and enter TOTP code generated from the two-factor Authentication app.
+![Setup User](../../img/quickstart/digitalocean/setup-user.png)
+
+Once you setup a password and provide a valid TOTP code, the user setup process will be complete, and you will be redirected to WebUI:
+
+![Teleport WebUI](../../img/quickstart/digitalocean/webui.png)
+
+Congrats! You've completed setting up Teleport! Finally, you are a step closer to managing secure access to your infrastructure hosted in DigitalOcean. 
+
+## Step 4/4. Protect access to DigitalOcean Services with Teleport!
+
+The next step for you should be to protect direct SSH access to your droplets using Teleport. You can also secure access to Database, Marketplace apps, and Kubernetes Cluster using Teleport. Below are the links to get started further:
```

Comment:
> I think the benefit of performing each next step should be more specific in relation to a user's DigitalOcean environment.

I agree. We need individual "how-to" guides specific to DigitalOcean. To begin with, even a simple "how to protect ssh access with Teleport" would suffice. Since we do not have those tutorials available, I thought the tiles should help user to navigate next.